client/my-sites/domains/domain-management/transfer/transfer-to-other-site/confirmation-dialog.jsx

```diff
@@ -29,7 +29,7 @@ class TransferConfirmationDialog extends PureComponent {
 		const targetSiteTitle = get( targetSite, 'title', translate( 'Site Title' ) );
 		if ( isMapping ) {
 			return translate(
-				'Do you want to transfer mapping of {{strong}}%(domainName)s{{/strong}} ' +
+				'Do you want to transfer domain connection of {{strong}}%(domainName)s{{/strong}} ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Do you want to transfer mapping of {{strong}}%(domainName)s{{/strong}} to site {{strong}}%(targetSiteTitle)s{{/strong}}?' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/domain-transfer-dialog-text)